packages/arb-avm-cpp/avm_values/src/pool.cpp

```diff
@@ -44,8 +63,28 @@ std::shared_ptr<RawTuple> TuplePool::getResource(size_t s) {
     return resource;
 }
 
-void TuplePool::returnResource(std::shared_ptr<RawTuple>&& object) {
-    if (!shuttingDown) {
-        resources[object->data.size()].push_back(std::move(object));
+void TuplePool::deleteTuple(UniqueTuple tup) {
+    if (shuttingDown) {
```

Comment:
Empty if statement shouldn't exist.  Should there be code inside?  Or should it be deleted?

Reply:
I removed the dead code